public/app/features/plugins/built_in_plugins.ts

```diff
@@ -35,7 +35,7 @@ import * as barGaugePanel from 'app/plugins/panel/bargauge/module';
 
 import * as exampleApp from 'app/plugins/app/example-app/module';
 
-const builtInPlugins = {
+const builtInPlugins: any = {
```

Comment:
not sure you need this, typescript should be able to figure out type from declaration?

Reply:
That's what I thought, but as it turns out noImplicitAny complains when trying to read an object property by passing a string as index `builtInPlugins[propertyString]`. The other option would be to do ts-ignore, but it feels dirty.

Reply:
Ok, let’s use any then